src/core/unit_tests/AutoParameter_test.cpp

```diff
@@ -24,6 +24,27 @@ BOOST_AUTO_TEST_CASE(direct_binding) {
   BOOST_CHECK(i == 42);
 }
 
+BOOST_AUTO_TEST_CASE(method_pointer) {
```

Comment:
I think this is not related to the PR, but if you want to keep it please describe what the test tests.

Reply:
Those are used in the SciptInterface of SimplePore.